HealthGPTTests/PromptGeneratorTests.swift

```diff
@@ -6,12 +6,14 @@
 // SPDX-License-Identifier: MIT
 //
 
-@testable import HealthGPT
-import XCTest
+import Foundation
 
-class PromptGeneratorTests: XCTestCase {
-    var sampleHealthData: [HealthData] = createSampleHealthData()
+@testable import HealthGPT
+import Testing
 
```

Comment:
Please keep two empty lines between import statements and the main content of the file.
```suggestion


```